### ISSUE TITLE
Error dialog "File not found 'Attack'" when ZAP started

### ISSUE THREAD

**Issue by pediRAM** [zaproxy/zaproxy#7430]
### Describe the bug

After starting ZAP an Error-Dialog with following message is shown "File not found 'Attack'":
(Please see uploaded screenshot!)

# Environment:
## Operating System: 
Windows 10 Pro Ver. 21H2
## Java Runtime:
openjdk version "1.8.0_302"
OpenJDK Runtime Environment (Temurin)(build 1.8.0_302-b08)
OpenJDK 64-Bit Server VM (Temurin)(build 25.302-b08, mixed mode)
## Type of start/call:
"zap.bat" in folder "C:\Program Files\OWASP\Zed Attack Proxy"

### Steps to reproduce the behavior

1. Goto folger: "C:\Program Files\OWASP\Zed Attack Proxy"
2. Type: "zap.bat" and press <ENTER>

### Expected behavior

Just a normal start without any errors.

### Software versions

ZAP: 2.11.1
Java Runtime: 
openjdk version "1.8.0_302"
OpenJDK Runtime Environment (Temurin)(build 1.8.0_302-b08)
OpenJDK 64-Bit Server VM (Temurin)(build 25.302-b08, mixed mode)
Windows: 10 Pro 21H2

### Screenshots

<img width="165" alt="2022-08-21 10_13_07-OWASP ZAP" src="https://user-images.githubusercontent.com/11803923/185782643-adecbb31-c7ed-4ccf-a6a7-9607b8d8e399.png">

### Errors from the zap.log file

[zap.log](https://github.com/zaproxy/zaproxy/files/9388726/zap.log)

### Would you like to help fix this issue?

- [X] Yes

**Comment by thc202** [comment]
Which command line arguments are you passing to ZAP?

**Comment by pediRAM** [comment]
> Which command line arguments are you passing to ZAP?

None.
Just calling "zap.bat" without any arguments.

**Comment by thc202** [comment]
Something is being passed to ZAP, the error indicates that "Attack" was passed as argument. Did you modify the bat file?

**Comment by kingthorin** [comment]
It’s probably Windows failing to properly handle a path with spaces.

**Comment by pediRAM** [comment]
> Something is being passed to ZAP, the error indicates that "Attack" was passed as argument. Did you modify the bat file?

No. I installed ZAP, then in console/cmd changed to the directory where "zap.bat" is saved, then typed "zap.bat" and hit the <ENTER> key. The file is still As it was installed. I even didn't open it in any editor.
If you want I can send you "zip.bat".
I did not pass any arguments, nor modified the batch script.

**Comment by kingthorin** [comment]
Ya it's a windows issue, if you'd used the short names in the path `C:\progra~1` etc it wouldn't error. (I'm 99% sure).

**Comment by thc202** [comment]
How is the path being passed to ZAP though?

**Comment by kingthorin** [comment]
I’d have to look at the batch script, however Windows handles pwd

**Comment by thc202** [comment]
https://github.com/zaproxy/zaproxy/blob/8587632dc945542586baf498f638aed687d20527/zap/src/main/dist/zap.bat#L1-L7

**Comment by kingthorin** [comment]
Might need to be quoted to properly handle the spaces. I’ll do some testing.

**Comment by kingthorin** [comment]
@pediRAM is there a particular reason you want to start zap via the bat?

**Comment by kingthorin** [comment]
Are you using the standard command prompt? Or powershell? Or terminal?

**Comment by kingthorin** [comment]
> Might need to be quoted to properly handle the spaces. I’ll do some testing.

I've been unable to re-create.

**Comment by thc202** [comment]
Closing, feel free to leave a comment providing the exact steps that reproduce the issue.

**Comment by pediRAM** [comment]
> @pediRAM is there a particular reason you want to start zap via the bat?

Yes. Because doubleclicking the ZAP shows a dialog, which says Java Runtime 1.8 is needed... so I am forced to start ZAP by "zap.bat".

By the way: I have already installed OpenJDK 18 (Community Edition), The "%JAVA_HOME%" environment variable is already set, and also "%PATH%" contains the "bin" directory of OpenJDK.

**Comment by thc202** [comment]
That's #5368, check this comment https://github.com/zaproxy/zaproxy/issues/5368#issuecomment-597182540

**Comment by pediRAM** [comment]
> Closing, feel free to leave a comment providing the exact steps that reproduce the issue.

Sorry for long delay.
The issue have been solved somehow/automatically... I cannot explain what produced the issue, and I cannot explain how it was solved. But, what I have observed: 
- The issue/error occured on the very first run (directly after installation).
- Then, ZAP updated some "scripts"/"modules" (I saw update-window)
- From the 2nd run on, the issue was solved (the "Attack" dialog was never shown again)!
IMHO: maybe one of the "modules" or "scripts" for attacking is the source of this particular issue, and when updated the issue cannot be reproduced.
Thank you all for your efforts.
